service-registry/src/main/java/io/servicecomb/serviceregistry/definition/MicroserviceDefinition.java

```diff
@@ -95,10 +95,8 @@ private void initCombinedFrom(List<ConfigModel> configModels) {
             Configuration conf = ConfigUtil.createConfig(Arrays.asList(model));
             String name =
                 conf.getString(DefinitionConst.qulifiedServiceNameKey, DefinitionConst.defaultMicroserviceName);
-            if (!StringUtils.isEmpty(name)) {
-                checkMicroserviceName(name);
-                combinedFrom.add(name);
```

Comment:
seems like there has no unit tests for this class and no test covers this scene.